configs/retinanet/retinanet_r18_fpn_1x8_1x_coco.py

```diff
@@ -0,0 +1,15 @@
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# data
+data = dict(samples_per_gpu=8)
+
+# optimizer
+model = dict(
+    backbone=dict(
+        depth=18,
+        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet18')),
+    neck=dict(in_channels=[64, 128, 256, 512]))
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
```

Comment:
Please comment why not 0.005.

Reply:
> Please comment why not 0.005.
I have tested Retinanet_r18 on COCO with lr equals 0.005, got 31.6 mAP in the end.
Should I change it to 0.005 follow the linear scaling rule ?

Reply:
Whichever is more performant,  and we can with a note.